deepinv/physics/compressed_sensing.py

```diff
@@ -51,6 +53,8 @@ class CompressedSensing(LinearPhysics):
     It is recommended to use ``fast=True`` for image sizes bigger than 32 x 32, since the forward computation with
```

Comment:
I would remove this 'fast' lines and point the user to the Structured random:

something like:

For image sizes bigger than 32 x 32 the forward computation can be prohibitively expensive due to its :math:`O(mn)` complexity. In this case, we recommend using :meth:`deepinv.physics.StructuredRandom` instead.